ffi/dotnet/Devolutions.Picky/src/Pem.Addons.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Runtime.InteropServices;
+
+using Devolutions.Picky.Diplomat;
+
+namespace Devolutions.Picky;
+
+public partial class Pem
+{
+	// TODO: maybe this should be part of the Diplomat namespace in DiplomatRuntime.cs
+#if __IOS__
+    private const string NativeLib = "libDevolutionsPicky.framework/libDevolutionsPicky";
+#else
+    private const string NativeLib = "DevolutionsPicky";
+#endif
+
+	/// Returned data should not be modified!
+	[DllImport(NativeLib, CallingConvention = CallingConvention.Cdecl, EntryPoint = "Pem_peek_data", ExactSpelling = true)]
+	internal static unsafe extern IntPtr PeekData(Raw.Pem* self, out nuint len);
+
+    public byte[] ToData()
+    {
+        unsafe
+        {
+            if (_inner == null)
+            {
+                throw new ObjectDisposedException("Pem");
```

Comment:
nit: `null` doesn't mean the object was disposed; maybe `NullReferenceException` or `ArgumentException` is more appropriate?

Reply:
Indeed! But, in this case it does, `_inner` is `null` only when `Pem` has been disposed of (see `Generated/Pem.cs`)

Reply:
Ah ok, great!